### PR TITLE
Fix errors in documentation

### PR DESCRIPTION
**Pull request by Tan-Jia-Rong**
## Changes made

1. Improve Javadoc comments and messages.
2. Update UserGuide.md to reflect changes in codes

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -403,10 +405,10 @@ OR
 ```
 find-person Alex
 ```
-In either of the above examples, we find every contact with names starting with Alex from Plannit.
+In either of the above examples, we find every person whose names starts with Alex in Plannit.
```
Comment (cheeheng) [inline docs/UserGuide.md:408]: Grammatical error in `starts`?

File: src/main/java/seedu/address/logic/commands/FindPersonCommand.java [hunk 2]
```diff
@@ -8,15 +8,15 @@
 import seedu.address.model.person.NameStartsWithKeywordPredicate;
 
 /**
- * Finds and lists all people in Plannit whose name starts with the argument keyword.
+ * Finds and lists all person in Plannit whose name starts with the argument keyword.
```
Comment (cheeheng) [inline src/main/java/seedu/address/logic/commands/FindPersonCommand.java:11]:
```suggestion
 * Finds and lists all persons in Plannit whose name starts with the argument keyword.
```

Another grammatical error?

File: src/main/java/seedu/address/model/module/ModuleTitle.java [hunk 3]
```diff
@@ -1,7 +1,7 @@
 package seedu.address.model.module;
 
 /**
- * Represents a {@code Module}'s title (optional field) in Plannit.
+ * Represents a {@code Module}'s title (optional field) in the address book.
```
Comment (cheeheng) [inline src/main/java/seedu/address/model/module/ModuleTitle.java:4]: Address book or Plannit? Let's discuss.

**Review by cheeheng — COMMENTED** [review]
LGTM except for minor nitpicks.

File: src/main/java/seedu/address/model/module/ModuleTitle.java (hunk 3 above)
Comment (Tan-Jia-Rong) [reply src/main/java/seedu/address/model/module/ModuleTitle.java:4]: I changed this to address book to align with the other classes within the module package. I feel that it is interchangeable to a small extent given that we are technically storing the data in an address book, whereby our application is called Plannit.

File: docs/UserGuide.md (hunk 1 above)
Comment (Tan-Jia-Rong) [reply docs/UserGuide.md:408]: You are right .-. I shall adopt the suggestion provided by grammarly :)
image

**Review by cheeheng — APPROVED** [review]
LGTM

**Review by teoyuqi — APPROVED** [review]
LGTM

**Pull request merged**